README.md

```diff
@@ -77,16 +81,17 @@ See [this issue](https://github.com/CodinGame/monaco-vscode-api/issues/186) or t
 The typescript language features extensions requires [SharedArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer) to enable project wide intellisense or only a per-file intellisense is provided.
 
 It requires [crossOriginIsolated](https://developer.mozilla.org/en-US/docs/Web/API/crossOriginIsolated) to be true, which requires assets files to be servers with some specific headers:
+
 - [Cross-Origin-Opener-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy): `same-origin`
 - [Cross-Origin-Embedder-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy): `require-corp` or `credentialless`
 
 At least thoses files should have the headers:
+
 - the main page html
 - the worker extension host iframe html: `webWorkerExtensionHostIframe.html`
 - the worker extension host worker javascript: `extensionHost.worker.js`
 
-If adding those headers is not an options, you can have a look at https://github.com/gzuidhof/coi-serviceworker
-
+If adding those headers is not an options, you can have a look at <https://github.com/gzuidhof/coi-serviceworker>
```

Comment:
I've realized it's not a good idea as it conflicts with the VSCode webview service worker :-\ maybe it's worth mentionning it while you're at it

Reply:
Do you just want to drop the sentence? You removed coi-serviceworker again completely here, didn't you?

Reply:
I've removed it because it conflicts with webviews, but not everyone is using webviews

Reply:
What about:

"If adding those headers is not an options, you can have a look at <https://github.com/gzuidhof/coi-serviceworker>, but only if you are not using webviews as it introduces problems then."

Reply:
Yep, seems good and not confusing (:

Reply:
👍

Reply:
Done ⬇️